src/zenml/cli/base.py

```diff
@@ -101,10 +94,15 @@ def clean(yes: bool = False) -> None:
 @cli.command("go")
 def go() -> None:
     """Quickly explore ZenML with this walkthrough."""
-    console.print(zenml_go_welcome_message, width=80)
-
+    from zenml.cli.text_utils import (
+        zenml_go_notebook_tutorial_message,
+        zenml_go_privacy_message,
+        zenml_go_welcome_message,
+    )
```

Comment:
Does this mean that the `zenml go` command will now be super slow?

Reply:
`zenml go` requires that you import `rich.Markdown`. The only change here is that the import is moved within the function. So, it will be imported only when you need it. The overall time for `zenml go` should stay the same.